_data/sw_clients.yml

```diff
@@ -634,37 +634,58 @@
           sts:
           userhost-in-names:
     - name: pounce (as Server)
-      # ref: https://git.causal.agency/pounce/tree/bounce.h#n63
+      # ref: https://git.causal.agency/pounce/about/pounce.1#Client_Configuration
       link: https://git.causal.agency/pounce/about/
       support:
         stable:
           account-notify:
+          account-tag: 1.2+
           away-notify:
+          batch: 1.2+
           cap-3.1:
+          cap-3.2: 1.2+
+          cap-notify: 1.2+
           chghost:
           extended-join:
           invite-notify:
+          labeled-response: 1.2+
+          message-tags: 1.2+
           monitor:
+          msgid: 1.2+
           multi-prefix:
           sasl-3.1:
+          sasl-3.2: 1.2+
           server-time:
+          setname: 1.2+
+          sts: 1.2+
           userhost-in-names:
         SASL:
           external:
     - name: pounce (as Client)
-      # ref: https://git.causal.agency/pounce/tree/bounce.h#n63
+      # ref: https://git.causal.agency/pounce/about/pounce.1#Client_Configuration
       link: https://git.causal.agency/pounce/about/
       support:
         stable:
           account-notify:
+          account-tag: 1.2+
           away-notify:
+          batch: 1.2+
           cap-3.1:
+          cap-3.2: 1.2+
+          cap-notify: 1.2+
           chghost:
           extended-join:
           invite-notify:
+          labeled-response: 1.2+
+          message-tags: 1.2+
           monitor:
+          msgid: 1.2+
           multi-prefix:
           sasl-3.1:
+          sasl-3.2: 1.2+
+          server-time: 1.2+
+          setname: 1.2+
+          sts: N/A (TLS-only)
```

Comment:
This should be marked N/A separately, as it's done over here: https://github.com/ircv3/ircv3.github.io/blob/master/_data/sw_servers.yml#L17

Reply:
ah, didn't see that that was a thing